gensim/csparse/psparse.pyx

```diff
@@ -0,0 +1,94 @@
+#-----------------------------------------------------------------------------
```

Comment:
Better to remove `README.*` files, but mention links to original site/repo/any useful information here.